kong/plugins/request-termination/daos.lua

```diff
@@ -0,0 +1,3 @@
+return {
+  tables = {"request_termination"}
+}
```

Comment:
this entire file is not needed, unless I'm missing something...

Kong will handle the plugin configuration including all fetching/storing of that data. The dao files (and also the migration files) are only required if the plugin has its own entities to store (for example the authentication plugins need to store credentials besides their own config)

so this can go, as well as the migrations files